cmd/generate.go

```diff
@@ -23,14 +35,17 @@ Usage of dynamic-buildkite-template
 This Program generates step for the provided plugins with configurations
 	`,
 	Run: func(cmd *cobra.Command, args []string) {
+		// check for overrides
+		ParseOverrides(g, cmd)
+		// load trivy plugin config
+		LoadTrivyConfigs()
+		// load cosign plugin config
+		LoadCosignConfigs()
+		// generate the build template
+		generator.GenerateBuildSteps(g, os.Stdout, util.TEMPLATE_FILE_PATH)
```

Comment:
the return value (error) should be checked

Reply:
Done